README.md

```diff
@@ -1,30 +1,68 @@
 # PUN-street-access
+
 PUA
 
+# how to run it
 
-# how to run it 
 ## frontend
+
 ### env
+
 ```
 cd frontend
 npm install
 ```
 
 ### run
+
 ```bash
 cd frontend
 npm run dev -- --open
 ```
+
 it will auto reload after code change
 
 ## backend
 
 ### env
 
 not yet use
+
 ```bash
+
 ```
+
 ### run
+
 ```bash
 go run .
```

Comment:
don't you need `go get`?

Reply:
it look no need now
maybe huhuhu pr will add it

Reply:
fixed